utils/common.py

```diff
@@ -24,3 +27,17 @@ def extract_zip_to_temp_dir(application_path):
         zip_ref.extractall(tempdir.name)
 
     yield tempdir.name
+
+def run_containerless_parametrize(func):
+    @pytest.mark.parametrize(
+        "additional_args",
+        [
+            {"--run-local=true": None},  # Running in container mode
+            {"--run-local=false": None}  # Running without container
```

Comment:
I thinks the comments are in the incorrect order.

When `run-local` is true it runs wuthout container.